docs/usage/shoot_status.md

```diff
@@ -95,8 +95,8 @@ The Shoot status also contains information about the last occurred error(s) (if
 
 Known error codes are:
 
-- `ERR_INFRA_UNAUTHORIZED` - indicates that the last error occurred due to invalid infrastructure credentials. It is classified as a non-retryable error code.
-- `ERR_INFRA_INSUFFICIENT_PRIVILEGES` - indicates that the last error occurred due to insufficient infrastructure privileges. It is classified as a non-retryable error code.
+- `ERR_INFRA_UNAUTHENTICATED` - indicates that the last error occurred due to client request not being completed because it lacks valid authentication credentials for the requested resource. It is classified as a non-retryable error code.
```

Comment:
```suggestion
- `ERR_INFRA_UNAUTHENTICATED` - indicates that the last error occurred due to the client request not being completed because it lacks valid authentication credentials for the requested resource. It is classified as a non-retryable error code.
```